src/main/kotlin/dev/arbjerg/ukulele/command/QueueCommand.kt

```diff
@@ -12,7 +12,7 @@ import org.springframework.stereotype.Component
 @Component
 class QueueCommand(
         private val players: PlayerRegistry
-) : Command("queue", "q", "list", "l") {
+) : Command("queue", "q", "list") {
```

Comment:
I stole the queue command's "l" shortcut for the Loop Command.  Because it already has "q" as a single character shortcut, I thought this would be ok.